### PR TITLE
feat: Add 2.1 Lootrun Missions [skip ci]

### PR DESCRIPTION
**Pull request by MakiilGrosnach**
Added 2.1 Mission choices:

HOARDER
JESTER'S TRICK
TREASURE HUNTING
GOURMAND
PORPHYROPHOBIA
CHRONOKINESIS
SAFETY SEEKER
WARMTH DEVOURER
COMPLETE CHAOS

### CODE REVIEW

**Review by ShadowCat117 — COMMENTED** [review]
Can you sort them to be alphabetical in their respective colours?

**Comment by kristofbolyai** [comment]
@ShadowCat117 Can you take over this PR for us in a few days? It may be that @MakiilGrosnach doesn't have Github notifications enabled. Either way, thank you for your contribution.

**Comment by ShadowCat117** [comment]
They did mention in the #wynntils-development channel on discord that they saw the notification but if it hasn't been done in a few days I can commit the change

**Comment by MakiilGrosnach** [comment]
Updated it, Sorry about the wait. Was working

File: common/src/main/java/com/wynntils/models/lootrun/type/MissionType.java [hunk 1]
```diff
@@ -13,17 +13,26 @@ public enum MissionType {
     FAILED("Failed", ChatFormatting.DARK_RED),
 
     CLEANSING_GREED("Cleansing Greed", ChatFormatting.YELLOW),
-    HIGH_ROLLER("High Roller", ChatFormatting.YELLOW),
+    HOARDER("Hoarder", ChatFormatting.YELLOW),
+    JESTERS_TRICK("Jester's Trick", ChatFormatting.YELLOW),
     MATERIALISM("Materialism", ChatFormatting.YELLOW),
+    HIGH_ROLLER("High Roller", ChatFormatting.YELLOW),
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/models/lootrun/type/MissionType.java]: This one needs to be moved

**Review by ShadowCat117 — APPROVED** [review]
Thanks!

**Pull request merged**